rezerwacje-front/src/views/Exams/components/ExamsContent/ExamsContent.jsx

```diff
@@ -33,11 +36,26 @@ const ExamsContent = () => {
         });
     };
 
-    const recruitmentSpaces = ["lato 2020", "zima 2021"];
+    useEffect(() => {
+        fetch("/api/recruitment/all")
+            .then(res => res.json())
+            .then(json => {
+                setRecruitments(json["recruitments"]);
+            })
+            .catch(e => console.log(e));
+    }, []);
 
-    const recruitmentPeriods = ['1', '2'];
+    useEffect(() => {
+        if (values.recruitment.length > 0)
```

Comment:
There's a problem, if only one value is available this never gets triggered and an exam cannot be added